### PR TITLE
Fix build windows

### PR DESCRIPTION
**Pull request by arielalmendral**

### CODE REVIEW

**Comment by arielalmendral** [comment]
- The current code does not compile with VS2012. Declaration fix in this PR.
- The code compiles with VS2015, possibly because it fully implements the C99 standard.

Do you still want to support VS2012?

File: devel/libecl/src/smspec_node.c [hunk 1]
```diff
@@ -737,9 +737,11 @@ smspec_node_type * smspec_node_alloc_lgr( ecl_smspec_var_type var_type ,
 }
 
 smspec_node_type* smspec_node_alloc_copy( const smspec_node_type* node ) {
-    if( !node ) return NULL;
 
     smspec_node_type* copy = util_malloc( sizeof * copy );
+
+    if( !node ) return NULL;
```
Comment (joakim-hove) [inline devel/libecl/src/smspec_node.c]: Thank you for the fix; I want to merge this - however the current fix will leak. I suggest instead:
```C
if (!node) return NULL;
{ 
     smspec_node_type * copy = util_malloc( sizeof * copy );
     ....
     ....
     return node;
}
}
```

**Comment by arielalmendral** [comment]
OK, well observed. Using your fix...

**Comment by joakim-hove** [comment]
Thank you - merging.

**Pull request merged**